architecture/README.md

```diff
@@ -33,7 +33,9 @@ types of roles. The TiDB community has the following organizations:
 
 ## PMC
```

Comment:
@gingerkidney Suggesting removing this file, as it conflicts with https://github.com/pingcap/community/pull/152

Reply:
I think we'd better keep a `README.md` file for a directory.

Reply:
Sorry, I mean revert/reset this change in this file for this PR. Not removing it...